launchers/ci-build.sh

```diff
@@ -9,7 +9,8 @@ source /environment.sh
 set -eux
 
 # build both PDF and HTML
-mkdir -p ${JB_PDF_OUT_DIR}
+# TODO: re-enable this
```

Comment:
I assume this should be re-enabled before merge.

Reply:
This was disabled by @afdaniele  because the pdf version took very long to generate.